src/pages/HomePage/components/Comments/Comments.tsx

```diff
@@ -1,9 +1,91 @@
 import './Comments.scss';
+import rightarrow from  '../Comments/CommentImages/arrow right.svg';
+import quote from  '../Comments/CommentImages/Vector.svg';
+import line from  '../Comments/CommentImages/Line 6.svg';
+
 
 export const Comments= () =>{
     return(
-        <>
-        <h1>Comments</h1>
-        </>
+        <div className="comments narrow-container">
+    <div className="mixin">
+       <p className="explanation">We cook – they enjoy every <br/> minute of their <img  className="rightarrow"  src={rightarrow}/>
+        lives</p>
+        <p className="because">Because we...</p>
+      <div className="quote">
+          <img src={quote}/>
+          <p className="quotation">We don't accept half-measures and  shortcuts, because what we care <br/> about  is the WOW effect. And all the 
+          luck that our efforts are not in <br/>  vain. We know this because we closely follow the opinions of our <br/>  consumers.</p>
+        </div> 
+    </div>
+ <div className="fourpart"> 
+    <div className="comment-box">
+    <div className="rating" data-total-value="0">
+            <div className="stars" data-item-value="5">★</div>
+            <div className="stars" data-item-value="4">★</div>
+            <div className="stars" data-item-value="3">★</div>
+            <div className="stars" data-item-value="2">★</div>
+            <div className="stars" data-item-value="1">★</div>
```

Comment:
for the stars you can create a Rating component that will have a number in props, and this component will return proper set of stars, according to that number